src/pages/index.js

```diff
@@ -159,7 +160,7 @@ const IndexPage = props => (
           <div class="col col-0 col-sm-2 col-md-3" />
           <div class="col col-12 col-sm-8 col-md-6">
             <div class="card">
-              <img class="card-img-top" src={Reacticon} alt="Sponsor Logo" />
+              <mg class="card-img-top" src={Reacticon} alt="Sponsor Logo" />
```

Comment:
@jackahl typo?

Reply:
@jackahl is that a typo? Do you get email notifications when I write something like this? :)

Reply:
@tisto yes was typo. is fixed And no I dont get Mails for some reason. Gonna look it up in my settings